var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -144,14 +144,18 @@ class Boost(Package):
             description="Augment library layout with versioned subdirs")
     variant('clanglibcpp', default=False,
             description='Compile with clang libc++ instead of libstdc++')
+    variant('numpy', default=False,
+            description='Build the Boost NumPy library (requires +python)')
 
     depends_on('icu4c', when='+icu')
     depends_on('python', when='+python')
     depends_on('mpi', when='+mpi')
     depends_on('bzip2', when='+iostreams')
     depends_on('zlib', when='+iostreams')
+    depends_on('py-numpy', when='+numpy')
 
     conflicts('+taggedlayout', when='+versionedlayout')
+    conflicts('+numpy', when='~python')
```

Comment:
Does it link to this library? Should this be `type=('build', 'run')`?

Reply:
I'm not sure it is needed at link time.

Reply:
I checked, it doesn't.

Reply:
In that case, can you add `type=('build', 'run')`?

Reply:
Done.